index.html

```diff
@@ -33,10 +33,10 @@
 
 
     <!--ID SECTION--> 
-    <main id="main">
+    <main id="main" class="background-main">
```

Comment:
Your desktop background doesn't quite look like the one from the template design.

Here's your design:

<img width="723" alt="Screen Shot 2022-06-02 at 13 34 21" src="https://user-images.githubusercontent.com/82426682/171701800-9f2ec616-6d6f-49fb-8ef6-cbeb23f45c3c.png">

Here's the template design:

<img width="861" alt="Screen Shot 2022-06-02 at 13 34 41" src="https://user-images.githubusercontent.com/82426682/171701851-0af9aed4-7f3d-482f-8ae5-505b0cb43f6f.png">

You're missing a portion of the background image in the lower portion of the section. Kindly fix this.

Additionally, your social media icons are too squeezed together, in the template design there is significantly more space in between them. Kindly fix the `social-network` class as well.